connectors/sources/microsoft_teams.py

```diff
@@ -266,6 +627,84 @@ def client(self):
             tenant_id, client_id, client_secret, username, password
         )
 
+    async def _consumer(self):
+        """Async generator to process entries of the queue
+
+        Yields:
+            dictionary: Documents from Microsoft Teams.
+        """
+        while self.tasks > 0:
+            _, item = await self.queue.get()
+
+            if isinstance(item, EndSignal):
+                self.tasks -= 1
+            else:
+                yield item
+
+    def verify_filename_for_extraction(self, filename):
+        attachment_extension = list(os.path.splitext(filename))
+        if "" in attachment_extension:
+            self._logger.debug(
+                f"Files without extension are not supported, skipping {filename}."
+            )
+            return
+        if "." not in filename:
+            self._logger.debug(
+                f"Files without extension are not supported by TIKA, skipping {filename}."
+            )
+            return
```

Comment:
```suggestion
        if "" in attachment_extension:
            self._logger.debug(
                f"Files without extension are not supported, skipping {filename}."
            )
            return
```
From what I can understand, if the `filename` is missing `.`, then `attachment_extension` will always have an empty value, so the second check here is redundant.

Sample testing:

```
>>> os.path.splitext("test.txt")
('test', '.txt')

>>> os.path.splitext("tmp/test.txt")
('tmp/test', '.txt')

>>> os.path.splitext("tmp.txt/test.txt")
('tmp.txt/test', '.txt')

>>> os.path.splitext(".txt")
('.txt', '')

>>> os.path.splitext(".")
('.', '')

>>> os.path.splitext("test")
('test', '')
```